Makefile

```diff
@@ -75,6 +75,13 @@ rebuild-expected-api-docs: run-app-bg
 	rm _new_api_docs.json
 	git diff ./conbench/tests/api/_expected_docs.py
 
+
+# Build HTML docs, exiting non-zero if there are warnings. Those can be very bad.
```

Comment:
While I agree that erroring on warnings in CI builds | deploys is good, not sure we want that when iterating locally. Could we make that an option?

Reply:
Done: removed it from the Makefile command since one can just look at the warnings locally